bundles/core-commands/commands/put.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+module.exports = (srcPath) => {
+  const Broadcast = require(srcPath + 'Broadcast');
+  const Parser = require(srcPath + 'CommandParser').CommandParser;
+  const ItemType = require(srcPath + 'ItemType');
+
+  return {
+    usage: 'put <item> <container>',
+    command : (state) => (args, player) => {
+      args = args.trim();
+
+      if (!args.length) {
+        return Broadcast.sayAt(player, 'Put what where?');
+      }
+
+      // put 3.foo in bar -> put 3.foo bar -> put 3.foo into bar
+      const parts = args.split(' ').filter(arg => !arg.match(/in/) && !arg.match(/into/));
+
+      if (parts.length === 1) {
+        return Broadcast.sayAt(player, "Where do you want to put it?");
+      }
+
+      let fromList = player.inventory, fromArg = parts[0], item = null, toContainer = null, toArg = parts[1];
+
+      toContainer = Parser.parseDot(toArg, player.room.items);
```

Comment:
Yep, and honestly once we've reached this point nothing changes (or should change) so mostly everything here can be const.